guacamole-common-js/src/main/webapp/modules/KeyEventInterpreter.js

```diff
@@ -96,6 +96,21 @@ Guacamole.KeyEventInterpreter = function KeyEventInterpreter(batchSeperation, st
          */
         this.modifier = template.modifier || false;
 
+        /**
+         * Whether this key should always be printed by name AND rendered
+         * directly, if directly typed. Any directly-typed key that has this
+         * flag set to true will result in the value of the key being rendered
+         * directly, followed by the name being printed as if it was a
+         * non-printable or non-typed character.
+         *
+         * If the key was not directly typed, this flag will have no effect on
+         * behavior.
+         *
+         * @type {!boolean}
+         * @default false
+         */
+        this.printAndRender = template.printAndRender || false;
```

Comment:
I'm on the fence regarding this being something `Guacamole.KeyEventInterpreter` should represent and decide. My initial thinking would be that this should be an implementation decision.

Thoughts?

Reply:
Never mind - I see this is `@private` _and_ only ever feeds into the `simpleText` convenience value. Implementations are still free to do whatever they wish.